docs/src/components/HomepageFeatures/index.js

```diff
@@ -49,10 +63,11 @@ export default function HomepageFeatures() {
   return (
     <section className={styles.features}>
       <div className="container">
-        <div className="row">
+        <div className="row ">
```

Comment:
```suggestion
        <div className="row">
```

idem la il te faut prettier